```
Error running mint automation
+ mkdir -p 5934-b70432f/gopath/src/github.com/minio
+ git -C 5934-b70432f/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 5934-b70432f/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 5934-b70432f/gopath/src/github.com/minio/minio fetch --quiet minio pull/5934/head:pr5934
+ git -C 5934-b70432f/gopath/src/github.com/minio/minio checkout --quiet pr5934
+ GOPATH=/home/bala/mintauto/5934-b70432f/gopath
+ make -C 5934-b70432f/gopath/src/github.com/minio/minio --quiet
# github.com/minio/minio/cmd
cmd/http-stats.go:194:2: undefined: httpRequests
cmd/metrics.go:143:27: undefined: httpRequests
make: *** [build] Error 2

```

--- a/cmd/http-stats.go
+++ b/cmd/http-stats.go
@@ -190,8 +190,6 @@ func (st *HTTPStats) updateStats(r *http.Request, w *httpResponseRecorder, durat
 			st.successDELETEs.Duration.Add(durationSecs)
 		}
 	}
-	// Increment the prometheus http request count with appropriate label
-	httpRequests.With(prometheus.Labels{"request_type": r.Method}).Inc()
 	// Increment the prometheus http request response histogram with appropriate label
 	httpRequestsDuration.With(prometheus.Labels{"request_type": r.Method}).Observe(durationSecs)
 }

--- a/cmd/metrics.go
+++ b/cmd/metrics.go
@@ -26,13 +26,6 @@ import (
 )
 
 var (
-	httpRequests = prometheus.NewCounterVec(
-		prometheus.CounterOpts{
-			Name: "minio_http_requests_total",
-			Help: "Total number of requests served by current Minio server instance",
-		},
-		[]string{"request_type"},
-	)
 	httpRequestsDuration = prometheus.NewHistogramVec(
 		prometheus.HistogramOpts{
 			Name:    "minio_http_requests_duration_seconds",
@@ -44,7 +37,6 @@ var (
 )
 
 func init() {
-	prometheus.MustRegister(httpRequests)
 	prometheus.MustRegister(httpRequestsDuration)
 	prometheus.MustRegister(newMinioCollector())
 }
@@ -107,6 +99,7 @@ func (c *minioCollector) Collect(ch chan<- prometheus.Metric) {
 		prometheus.CounterValue,
 		float64(globalConnStats.getTotalInputBytes()),
 	)
+
 	// Total/Free Storage Bytes
 	ch <- prometheus.MustNewConstMetric(
 		prometheus.NewDesc(
@@ -123,15 +116,6 @@ func (c *minioCollector) Collect(ch chan<- prometheus.Metric) {
 			nil, nil),
 		prometheus.GaugeValue,
 		float64(s.Free),
-	)
-	// Set Server Start Time
-	ch <- prometheus.MustNewConstMetric(
-		prometheus.NewDesc(
-			prometheus.BuildFQName("minio", "server", "start_time_seconds"),
-			"Time when current Minio server instance started",
-			nil, nil),
-		prometheus.GaugeValue,
-		float64(globalBootTime.Unix()),
 	)
 
 	// Minio Total Disk/Offline Disk
@@ -156,10 +140,7 @@ func (c *minioCollector) Collect(ch chan<- prometheus.Metric) {
 func metricsHandler() http.Handler {
 	registry := prometheus.NewRegistry()
 
-	err := registry.Register(httpRequests)
-	logger.LogIf(context.Background(), err)
-
-	err = registry.Register(httpRequestsDuration)
+	err := registry.Register(httpRequestsDuration)
 	logger.LogIf(context.Background(), err)
 
 	err = registry.Register(newMinioCollector())

